src/background/utils.ts

```diff
@@ -37,20 +37,24 @@ export async function getTabIdWithSearchOpen(
 export async function getTabsInCurrentWindow() {
   // should it return the current tab??
   const tabs = await browser.tabs.query({ currentWindow: true });
-  return tabs
-    .filter((tab) => {
-      // will all pages have a title?
-      return tab.id && tab.id !== browser.tabs.TAB_ID_NONE && tab.url;
-    })
-    .map(({ id, favIconUrl, title, url }) => {
+  const results: TabData[] = [];
+  const tabNum = tabs.length;
+
+  for (let i = 0; i < tabNum; i++) {
+    const tab = tabs[i];
+    // will all pages have a title?
```

Comment:
From what I have tested, yes. If no title in the HTML is provided, it defaults to its URL. The only reason it won't be present in this object is if we aren't using the right permissions (which we are) https://developer.chrome.com/docs/extensions/reference/tabs/#type-Tab